### PR TITLE
added a note to SXTMap about the coordinate frame

### PR DESCRIPTION
**Pull request by nabobalis**
Trying to address https://github.com/sunpy/sunpy/pull/6436#issuecomment-1264174932

I also added the comment from Jack about the reprocess date in 2016.

### CODE REVIEW

File: sunpy/map/sources/yohkoh.py [hunk 1]
```diff
@@ -33,6 +33,14 @@ class SXTMap(GenericMap):
     Yohkoh was launched on 30 August 1991 and ceased operations on
     14 December 2001.
 
+    Notes
+    -----
+    Observer location:  We use the ITRS coordinates provided in the FITS header (``LAT``, ``LONG``, ``RADIUS``)
+    for the spacecraft location even when coordinates in other frames are provided (``HGS_LON``, ``HGS_LAT``,``DSUN_OBS``).
```
Comment (dstansby) [inline sunpy/map/sources/yohkoh.py:39]:
```suggestion
    for the spacecraft location even when coordinates in a Heliographic frame are provided (``HGS_LON``, ``HGS_LAT``,``DSUN_OBS``).
```

File: sunpy/map/sources/yohkoh.py [hunk 2]
```diff
@@ -33,6 +33,14 @@ class SXTMap(GenericMap):
...
+    The two frames differ substantially and it the ITRS coordinates are more accurate than the HGS information.
```
Comment (dstansby) [inline sunpy/map/sources/yohkoh.py:40]:
```suggestion
    The two frames differ substantially and the ITRS coordinates are more accurate than the HGS information.
```

File: sunpy/map/sources/yohkoh.py [hunk 3]
```diff
@@ -33,6 +33,14 @@ class SXTMap(GenericMap):
...
+
+    These changes were required due to the FITS files being re-processed in 2016.
```
Comment (dstansby) [inline sunpy/map/sources/yohkoh.py:42]:
```suggestion
    Using this information for the observer coordinate was implemented in sunpy 4.0.6 to work with the FITS files that were re-processed in 2016.
```

File: sunpy/map/sources/yohkoh.py [hunk 4]
```diff
@@ -33,6 +33,14 @@ class SXTMap(GenericMap):
     Yohkoh was launched on 30 August 1991 and ceased operations on
     14 December 2001.
 
+    Notes
+    -----
+    Observer location:  We use the ITRS coordinates provided in the FITS header (``LAT``, ``LONG``, ``RADIUS``)
+    for the spacecraft location even when coordinates in a Heliographic frame are provided (``HGS_LON``, ``HGS_LAT``,``DSUN_OBS``).
+    The two frames differ substantially and the ITRS coordinates are more accurate than the HGS information.
```
Comment (ayshih) [inline sunpy/map/sources/yohkoh.py:40]:
```suggestion
    The two locations differ substantially, and the ITRS coordinates are more accurate than the HGS information.
```

**Review by ayshih — CHANGES_REQUESTED** [review]
Also:

- Wrap the lines in the docstring.  It can be viewed separate from Sphinx rendering.
- Edit the commit message to not have the phrase "bad note".  I think GitHub preserves those messages even if the PR is squash merged?

File: sunpy/map/sources/yohkoh.py (hunk 4 above)
Comment (nabobalis) [reply sunpy/map/sources/yohkoh.py]: Done

**Comment by nabobalis** [comment]
> Also:
> 
> * Wrap the lines in the docstring.  It can be viewed separate from Sphinx rendering.
> * Edit the commit message to not have the phrase "bad note".  I think GitHub preserves those messages even if the PR is squash merged?

Done both.

File: sunpy/map/sources/yohkoh.py [hunk 5]
```diff
@@ -33,6 +33,18 @@ class SXTMap(GenericMap):
     Yohkoh was launched on 30 August 1991 and ceased operations on
     14 December 2001.
 
+    Notes
+    -----
+    Observer location:  We use the ITRS coordinates provided in
+    the FITS header (``LAT``, ``LONG``, ``RADIUS``)
+    for the spacecraft location even when coordinates in a
```
Comment (ayshih) [inline sunpy/map/sources/yohkoh.py:40]:
```suggestion
    for the spacecraft location even when coordinates in the heliographic
```

File: sunpy/map/sources/yohkoh.py [hunk 6]
```diff
@@ -33,6 +33,18 @@ class SXTMap(GenericMap):
...
+    Heliographic frame are provided (``HGS_LON``, ``HGS_LAT``,``DSUN_OBS``).
```
Comment (ayshih) [inline sunpy/map/sources/yohkoh.py:41]:
```suggestion
    Stonyhurst frame are provided (``HGS_LON``, ``HGS_LAT``, ``DSUN_OBS``).
```
I believe the lack of a space after the comma messed up the rendering

**Review by ayshih — APPROVED** [review]
Minor fixes, otherwise good to go

File: sunpy/map/sources/yohkoh.py (hunk 6 above)
Comment (nabobalis) [reply sunpy/map/sources/yohkoh.py:41]: I missed that!

**Review by dstansby — APPROVED** [review]

**Comment by dstansby** [comment]
Good to merge if/when the doc bulid succeeds

**Comment by nabobalis** [comment]
Note looks good to me on RTD, so I will merge.

**Pull request merged**